### DISCUSSION TITLE
No support for pytest 7.4.0 yet?

### DISCUSSION THREAD

**Discussion by balazs-ita-epam** [coady/pytest-parametrized#12] [Q&A]
In a test automation framework we are using Python 3.11 and pytest 7.4.0. 

We would like to use parameterized test runs, but not the builtin pytest.mark.parametrize lib, we know the pytest-parametrized lib from other project. 

However, according to our tests the current pytest-parametrized does not support the pytest 7.4.0 yet.

Is this something to be expected soon? Any workaround?

**Comment by coady** [comment]
It supports the latest pytest. What issues are you having?
